index.js

```diff
@@ -54,4 +54,41 @@ function createList() {
     const element = document.querySelector('.main')
     element.appendChild(newList)
 
+    // ********************* adding a title for movie in the list *************************** //
+
+    // adding an event listener to this created inputBtn that is inside the beginning binge list box
+inputBtn.addEventListener('click', (e) => {
```

Comment:
I wonder if having the callback function as an arrow function inside the other one may make it harder to test for later...? I think i remember Oli saying it's good to break functions up to smaller functions if possible. Sorry, I don't know if that's super clear (or even correct!)

Reply:
This was a way that I am used to writing event listeners with the arrow function, and a challenge that I got this handler to work was by placing this inside the other, if this causes issues then I can modify this once we create the tests, these are all valid point's, thank you for the feedback ✨